Makefile

```diff
@@ -0,0 +1,9 @@
+PKG:=github.com/docker/docker-network
+PKG_PATH:=/go/src/$(PKG)
+RUN_CMD:=docker run --privileged --rm -v "$(shell pwd)":$(PKG_PATH) -w $(PKG_PATH) golang:1.4 go
+
+build:
```

Comment:
@mrunalp You can do locally with just go tools. Makefile will be for docker env.